idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -1165,6 +1224,8 @@ def run_stage2(lazy_run=True):
                                  AWS_BATCH_JOB_ID)
     configure_logger(log_file)
 
+    write_to_log("Starting stage...")
```

Comment:
where was this going before?

Reply:
Hah I added it last week but it didn't do anything if above configure_logger